### PR TITLE
Give Surbhi admin access to `koku-metrics-operator` namespace

### PR DESCRIPTION
**Pull request by naved001**
This PR creates a role binding and a group to accomplish this.

@larsks does this look right?

### CODE REVIEW

File: cluster-scope/overlays/ocp-staging/kustomization.yaml [hunk 1]
```diff
@@ -22,6 +24,7 @@ resources:
 patches:
   - path: apiservers/cluster_patch.yaml
   - path: oauths/cluster_patch.yaml
+  - path: groups/koku-metrics-operator-admin.yaml
```
Comment (larsks) [inline cluster-scope/overlays/ocp-staging/kustomization.yaml:27]: If this is a patch, then you need to include the group in the `resources` section.

**Review by larsks — CHANGES_REQUESTED** [review]
Looks fine except for an issue with the group I noted in a comment.

File: cluster-scope/overlays/ocp-staging/kustomization.yaml (hunk 1 above)
Comment (naved001) [reply cluster-scope/overlays/ocp-staging/kustomization.yaml:27]: not sure which group you are talking about. I included `../../base/user.openshift.io/groups/koku-metrics-operator-admin` in resources in this PR.

Comment (larsks) [reply cluster-scope/overlays/ocp-staging/kustomization.yaml:27]: But it's not listed in the `cluster-scope/overlays/ocp-staging/kustomization.yaml`, so there's nothing to patch.

Comment (larsks) [reply cluster-scope/overlays/ocp-staging/kustomization.yaml:27]: I think I am simply blind.

**Review by larsks — APPROVED** [review]

**Comment by naved001** [comment]
haha no worries. I just realized does @skanthed have an account on ocp-staging cluster to begin with?

**Comment by larsks** [comment]
>  I just realized does @skanthed have an account on ocp-staging cluster to begin with?

Not yet, but that won't prevent us from deploying the rolebinding.

**Comment by larsks** [comment]
@skanthed you should log into https://console-openshift-console.apps.ocp-staging.massopen.cloud/dashboards which will create an account and we'll ensure you have appropriate access.

**Comment by skanthed** [comment]
@naved001 I logged in to the staging environment. Now you can give me the appropriate access I guess.

**Pull request merged**